.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,27 +1,38 @@
 ---
 name: Feature request
-about: Suggest an idea for this project
+about: Suggest an idea for a new feature
 title: ""
-labels: "enhancement"
+labels: "new feature"
 assignees: ""
 ---
 
-**Search terms you've used**
+<!-- 
+Thank you for reporting an issue.
 
+Please fill in as much of the template below as you're able. Feel free to delete any section you want to skip. 
+-->
+
+### Search terms you've used
 <!-- What search terms have you used to check whether this feature has been requested before? -->
 
-**Is your feature request related to a problem? Please describe.**
+## Suggestion
+
+<!-- A summary of what you'd like to see added or changed -->
 
-<!-- A clear and concise description of what the problem is. Ex. I'm always frustrated when [...] -->
+## Expected functionality/enhancement
 
-**Describe the solution you'd like**
+<!-- How should the new feature work -->
```

Comment:
To give us some leeway in the implementation, i.e. if there are issues with the proposed solution in the context of the larger codebase, but an alternative solution would still satisfy their use case.

```suggestion
<!-- How would you like the new feature work -->
```